### ISSUE TITLE
minikube start: VM IP address not resolving on v.0.15

### ISSUE THREAD

**Issue by Rurouni** [kubernetes/minikube#1000]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG

**Minikube version** (use `minikube version`): v0.15.0

**Environment**:
- **OS**:  Win10(1607)
- **VM Driver**: hyper-V
- **Docker version**: 1.13.0-rc6, build 2f2d055
- **Install tools**: minikube
- **Others**: none

**What happened**:
After switching to minikube 0.15 creating VM from scratch hangs in a loop trying to get VM IP address, logs keep writing that:`C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -NonInteractive (( Get-VM minikube ).networkadapters[0]).ipaddresses[0]`
Full command that I am running: 
`minikube start --kubernetes-version="v1.5.1" --vm-driver="hyperv" --show-libmachine-logs --alsologtostderr --hyperv-virtual-switch "Minikube" --memory 4096 --v=7`
Hyper-v switch is configured as internal with internet sharing and
same works if I delete VM and return back to minikube 0.14, so I think it's something to do with the new iso 
**What you expected to happen**:
minikube start works

**How to reproduce it** (as minimally and precisely as possible):
1) delete minikube VM and .minikube folder if running
2) minikube start --kubernetes-version="v1.5.1" --vm-driver="hyperv" --show-libmachine-logs --alsologtostderr --hyperv-virtual-switch "Minikube" --memory 4096 --v=7`

**Comment by BillyD73** [comment]
I am getting the same issue, Win 10 hyperv newest docker and kube

**Comment by rikbosch** [comment]
I'm experiencing the same issue. ifconfig eth0 shows an ip is being assigned by dhcp in the vm.

**Comment by Ohmen** [comment]
I am getting the same issue.
Additionl information: 

- Host can ping minikube vm and vice versa
- Vm has internet access

Environment:

- OS: Win10(1511)
- VM Driver: hyper-V
- Docker version: 1.13.0-beta39 (9861)
- Install tools: minikube

**Comment by Alexander-Kiselyov** [comment]
Same issue, reproduced with both fabric8's [gofabric8 tool](https://fabric8.io/guide/getStarted/gofabric8.html) (v0.4.113) and Minikube launcher itself (Win10, Hyper-V).
VM starts, I'm able to connect to it with Hyper-V Manager, but (( Get-VM minikube ).networkadapters[0]).ipaddresses[0] yields nothing.

**Comment by Rurouni** [comment]
I have just tested v0.16 with VSwitch set to external network instead of internal with internet sharing and it worked fine, so I am closing this issue

**Comment by fenkner** [comment]
I was having this problem, pulling my hair out and none of the explanations were helping me until I noticed @Alexander-Kiselyov last point in his posting, "Another viable option".  I followed that link then I realized what I had done wrong.  When you setup a virtual switch make sure you set it to PRIVATE and then inside that switch select EXTERNAL.  I was selecting external virtual switch with external network and this did not work.  I followed the rest of the instructions by deleting my minikube vm and blowing away the .minikube folder in my home directory.  I then ran minikube start and it all worked perfectly!  Finally!!  Thank you @Alexander-Kiselyov and @aaron-prindle !!
